index.html

```diff
@@ -1,222 +1,267 @@
-                <div class="lists">
-                    <hr id="second-line">
-                    <ul>
-                        <li class="bold">
-                            <img src="images/Rectangle 55.png" alt="rhombus">
-                            Language
-                        </li>
-                        <ul class="skills">
-                            <li>Javascript</li>
-                            <li>Ruby</li>
-                            <li>Html</li>
-                            <li>Css</li>
-                        </ul>
-                    </ul>
-                    <ul>
-                        <li class="bold">
-                            <img src="images/Rectangle 56.png" alt="rectangle">
-                            Frameworks
-                        </li>
-                        <ul class="skills">
-                            <li>Bootstrap</li>
-                            <li>Ruby on Rails</li>
-                            <li>RSPec</li>
-                            <li>CapyBara</li>
-                            <li>Selenium</li>
-                        </ul>
+                </ul>
+                <ul>
+                    <li class="bold">
+                        <img src="images/Rectangle 56.png" alt="rectangle">
+                        Frameworks
+                    </li>
+                    <ul class="skills">
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
                     </ul>
-                    <ul>
-                        <li class="bold">
-                            <img src="images/Rectangle 57.png" alt="circle">
-                            Skills
-                        </li>
-                        <ul class="skills">
-                            <li>Codekit</li>
-                            <li>Githubt</li>
-                            <li>Codepen</li>
-                            <li>Gitlab</li>
-                            <li>Terminal</li>
-                        </ul>
+                </ul>
+                <ul>
+                    <li class="bold">
+                        <img src="images/Rectangle 57.png" alt="circle">
+                        Skills
+                    </li>
+                    <ul class="skills">
+                        <li>Codekit</li>
+                        <li>Githubt</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
                     </ul>
+                </ul>
+            </div>
+        </section> <!-- end about me section -->
+
+        <!-- start contact form section -->
+        <section class="contact flex" id="contact">
+            <h3>
+                I'm always interested in hearing about new projects, so if you'd like
+                to chat please get in touch.
+            </h3>
```

Comment:
- Hey, the text content here should be a paragraph and not a header. let current this to improve our markup